grammar.js

```diff
@@ -339,7 +356,11 @@ module.exports = grammar({
 function stringBody (open, close, insert) {
   var contents = [ /\\./, RegExp('[^\\\\\\' + close + ']') ];
   if (typeof insert !== 'undefined') contents.push(insert);
-  return seq(open, repeat(choice.apply(null, contents)), token(prec(PREC.LITERAL, close)));
+  return seq(
+    (typeof open === 'string') ? token(prec(PREC.LITERAL, open)) : open,
```

Comment:
This ternary is a temporary hack to work around a tree-sitter bug when handling `token(blank())`.